Corona-Warn-App/src/main/java/de/rki/coronawarnapp/transaction/RetrieveDiagnosisKeysTransaction.kt

```diff
@@ -119,16 +125,29 @@ object RetrieveDiagnosisKeysTransaction : Transaction() {
     /** atomic reference for the rollback value for created files during the transaction */
     private val exportFilesForRollback = AtomicReference<List<File>>()
 
+    private val progressTowardsQuotaForRollback = AtomicReference<Int>()
+
     private val transactionScope: TransactionCoroutineScope by lazy {
         AppInjector.component.transRetrieveKeysInjection.transactionScope
     }
 
+    private const val QUOTA_RESET_PERIOD_IN_HOURS = 24
+
+    private val quotaCalculator: QuotaCalculator<Int> = GoogleQuotaCalculator(
+        incrementByAmount = 14,
```

Comment:
Does it still increment count by 14 each time `executeAPISubmission` is called? I thought that after https://github.com/corona-warn-app/cwa-app-android/pull/1088 it should increment it only by 1, as `provideDiagnosisKeys` is called once. Assuming that Google docs are correct:

> Calls to this method are limited to six per day for token=TOKEN_A and 20 per day when token != TOKEN_A (Allowlisted accounts are allowed 1,000,000 calls per day.)
>
> This quota applies per call, not per file or batch of files uploaded. In other words, a single call with multiple files or multiple batches counts as only one call. Days are defined as midnight to midnight of a device's UTC.

Reply:
This will be done in a separate PR as we plan to provide this change here before 1.4 official release. this way we single out the commit. But you are correct, with the changes in v1.5 mode, this needs to be set to 1